src/mach/imports.rs

```diff
@@ -195,9 +216,9 @@ impl<'a> BindInterpreter<'a> {
 	            // throwBadBindingAddress(address, segmentEndAddress, segmentIndex, start, end, p);
 	            // (this->*handler)(context, address, type, symbolName, symboFlags, addend, libraryOrdinal, "", &last);
 	            // address += sizeof(intptr_t);
+                    imports.push(Import::new(&bind_info, libs, segments, start_of_sequence));
                     let seg_offset = bind_info.seg_offset.wrapping_add(ctx.size() as u64);
                     bind_info.seg_offset = seg_offset;
-                    imports.push(Import::new(&bind_info, libs, segments));
```

Comment:
I fell into this rabbit hole while building a tool that uses `goblin` to explore [C++ RTTI structs](http://www.swag.uwaterloo.ca/acd/docs/ItaniumC++ABI.htm#rtti-layout). I got it working on statically linked binaries, but it was failing on Mac dynamically linked ones.

The RTTI structs have different data members depending on their type, and their type can be distinguished based on their vtable pointers. The failing types should have been importing the vtable of e.g. `abi::__class_type_info` from `libstdc++` -- the imports just weren't there in `goblin`, but I was seeing them at runtime and in a disassembler, which pointed to a `goblin` issue.

That finding prompted me to build a tool to dump Mach-O imports from `goblin`. This tool let me bug-hunt this issue, which ultimately turned into what landed in this PR :-)

Incidentally, `goblin::mach` vtable imports were slightly broken for a separate reason: C++ vtable pointers need to point [after the typeinfo pointer](http://www.swag.uwaterloo.ca/acd/docs/ItaniumC++ABI.htm#vtable), while on Mac the vtable symbols point at the start of the table. The `addend` field accounts for that offset. This differs from e.g. Windows where the vtable symbols point directly to where the vtable pointers need to point, even though those addresses may literally be past the end of the vtable.